### DISCUSSION TITLE
Support for "next third" from Spectacle

### DISCUSSION THREAD

**Discussion by gsinti** [rxhanson/Rectangle#329] [Ideas]
First of all, thanks for continuing to support and maintain this application! It's incredibly useful and makes window management a breeze, and the added support for Big Sur and Apple Silicon is fantastic.

I realize support for the feature mentioned in the subject was replaced with more explicit management of thirds. Was there a specific reason for this?

I liked the simplicity of using a single hotkey to cycle through thirds in Spectacle. Can this be brought into Rectangle?

Thanks!

**Comment by rxhanson** [comment]
This is already included in Rectangle as repeated executions of thirds actions. It could be explained in the UI, but most users that come from Spectacle are already used to undocumented repeated execution behavior for other actions (and it would be yet another thing to localize).

**Reply by gsinti** [reply]
Thanks for the tip! I had no idea. I just tried it now by repeating the hotkey for "First Third" and it's exactly what I was looking for!
